types/knockout/test/index.ts

```diff
@@ -1,4 +1,4 @@
-
+/// <reference types="knockout.postbox" />
```

Comment:
was it intentional? I don't see it's used. even if it was, need to avoid coupling of components unless they really depend.